source/_integrations/androidtv_remote.markdown

```diff
@@ -0,0 +1,382 @@
+---
+title: Android TV Remote
+description: Instructions on how to integrate Android TV remotes into Home Assistant.
+ha_category:
+  - Remote
+ha_release: 2023.4
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@tronikos'
+ha_quality_scale: platinum
+ha_domain: androidtv_remote
+ha_zeroconf: true
+ha_platforms:
+  - diagnostics
+  - remote
+ha_integration_type: device
+---
+
+The `androidtv_remote` platform allows you to control an Android TV device by sending [commands](https://github.com/tronikos/androidtvremote2/blob/main/TvKeys.txt) and launching apps.
+
+{% include integrations/config_flow.md %}
+
+## Entity
+
+This integration adds a `remote` entity which turns on/off the Android TV device.
+The entity has the `current_activity` attribute that shows the current foreground app on the Android TV.
+
+## Services
+
+### Service `remote.turn_off`
+
+Turn off the Android TV.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            |       no | Entity ID to target.
+
+### Service `remote.turn_on`
```

Comment:
This isn't a service provided by this integration, but by the `remote` integration.
Let's remove it here, as we do not need to create duplicate documentation.

Reply:
Resolving and keeping the other comment thread open. Depending on feedback there I can do the same for these sections too.